public/content/eips/index.md

```diff
@@ -54,10 +54,20 @@ Full standardization process alongside with chart is described in [EIP-1](https:
 
 If you’re interested to read more about EIPs, check out the [EIPs website](https://eips.ethereum.org/) and [EIP-1](https://eips.ethereum.org/EIPS/eip-1). Here are some useful links:
 
+### Official EIP resources
+
 - [A list of every EIP](https://eips.ethereum.org/all)
 - [A description of all EIP types](https://eips.ethereum.org/EIPS/eip-1#eip-types)
 - [A description of all EIP statuses](https://eips.ethereum.org/EIPS/eip-1#eip-process)
 
+### Community education projects {#community-projects}
+
+- [EIPs For Nerds](https://ethereum2077.substack.com/t/eip-research) — *EIPs For Nerds provides comprhensive, ELI5-style overviews of various Ethereum Improvement Proposals (EIPs), including core EIPs and application/infrastructure-layer EIPs (ERCs), to educate readers and shape consensus around proposed changes to the Ethereum protocol.* 
+- [PEEPanEIP](https://www.youtube.com/playlist?list=PL4cwHXAawZxqu0PKKyMzG_3BJV_xZTi1F) — *PEEPanEIP is an educational video series that discusses Ethereum Improvement Proposal (EIPs) and key features of upcoming upgrades.*
+- [EIPs.wtf](https://www.eips.wtf/) — *EIPs.wtf provides extra information for Ethereum Improvement Proposals (EIPs), including their status, implementation details, related pull requests, and community feedback.* 
+- [EIP.Fun](https://eipfun.substack.com/) — *EIP.Fun provides the latest news on Ethereum Improvement Proposals (EIPs), updates on EIP meetings, and more.*
+- [EIPs Insight](https://eipsinsight.com/) — *EIPs Insight is a representation of state of Ethereum Improvement Proposals (EIPs) process & statistics as per information collected from different resoucrces.*
+
```

Comment:
IMO, any public project should not require "sign-in" in general. I am not comfortable with a link with "Substack". However, I am okay with other community projects without subscription suggestions.

Reply:
> any public project should not require "sign-in" in general.

I'm not sure I understand what you mean here. Readers have the option to read articles on Substack without signing in (even if the publication is paid, only the paid articles requiring with a subscriber account to read the content). But if the issue is that readers on Substack are prompted with subscription suggestions (even though subscribing costs nothing and is only a way for readers to keep up with newly published content), how is that a problem?

> I am not comfortable with a link with "Substack".

Is there a reason Substack links shouldn't be included here? One of the reasons I separated the eips.ethereum.org website links from the rest was to ensure readers always go the authoritative source (eips.ethereum.org) for information about EIPs before considering visiting other platforms that also provide EIP-related information. But it doesn't mean the other platforms aren't as important: not every person who lands on the page is an expert capable of understanding the (often highly) technical descriptions of EIPs on eips.ethereum.org, so providing alternative educational resources for beginner/intermediate-level readers is important.